model/guest/GuestTestUser.php

```diff
@@ -37,17 +37,21 @@ public function __construct()
         $this->uri = \common_Utils::getNewUri();
     }
 
-    public function getIdentifier()
+    public function getIdentifier(): string
     {
         return $this->uri;
     }
 
-    public function getPropertyValues($property)
+    public function getPropertyValues($property): array
     {
+        if ($property === GenerisRdf::PROPERTY_USER_UILG && defined('DEFAULT_ANONYMOUS_INTERFACE_LANG')) {
```

Comment:
This is quite a hack.
Any way to let `UserLanguageServiceInterface` implementation in https://github.com/oat-sa/generis/pull/910 know about the fact the test session is a guest one, so that it could chose between the `DEFAULT_LANG` and `DEFAULT_ANONYMOUS_INTERFACE_LANG`?

Reply:
On the other hand `core_kernel_users_GenerisUser` uses a very similar approach.
All right, fair enough.

Reply:
I agree, not that elegant solution as ideally the default language should be defined according to the user type, yet, I think it's a bit better than creating a dependency with an implementation detail in the `UserLanguageServiceInterface`, wdyt?